src/Sentry/TransactionTracer.cs

```diff
@@ -253,7 +253,7 @@ public void Finish(Exception exception, SpanStatus status)
             Finish(exception, SpanStatusConverter.FromException(exception));
 
         /// <inheritdoc />
-        public ISpan? GetLastActiveSpan() => Spans.LastOrDefault(s => !s.IsFinished);
+        public ISpan? GetLastActiveSpan() => Spans.OrderBy(x => x.StartTimestamp).LastOrDefault(s => !s.IsFinished);
```

Comment:
```suggestion
        public ISpan? GetLastActiveSpan() =>
            // We need to sort by timestamp because the order of ConcurrentBag<T> is not deterministic
            Spans.OrderBy(x => x.StartTimestamp).LastOrDefault(s => !s.IsFinished);
```

Reply:
Sure, I've updated the CHANGELOG , thanks!

Reply:
Might turn out to be the same at runtime but arguably this is easier to reason about:

```suggestion
        public ISpan? GetLastActiveSpan() => Spans.OrderByDescending(x => x.StartTimestamp).FirstOrDefault(s => !s.IsFinished);
```